services/web/server/tests/integration/fixtures/celery_service.py

```diff
@@ -13,8 +13,8 @@
 
 
 @pytest.fixture(scope="module")
-def celery_service(app_config, docker_stack):
-    cfg = app_config["rabbit"]
+def celery_service(webserver_dev_config, docker_stack):
+    cfg = deepcopy(webserver_dev_config["rabbit"])

```

Comment:
ok, can we do _webserver_dev_config maybe? then it would be a bit clearer that one should not use it in normal tests.